lib/view/drawing_canvas/widgets/canvas_side_bar.dart

```diff
@@ -308,6 +330,39 @@ class CanvasSideBar extends HookWidget {
     }
   }
 
+  Future<Image> get _getImage async {
```

Comment:
I think this should be `ui.Image` to address the first review comment

Reply:
This has been resolved in the recent push